### ISSUE TITLE
Как запретить логиниться всем пользователям, кроме предопределённых?

### ISSUE THREAD

**Issue by werman** [LTD-Beget/sprutio#11]
Сейчас заходить можно из-под любого пользователя, что на мой взгляд не всегда хорошо. При беглом просмотре интерфейса/кода я не нашёл ничего что реализует этот функционал.

**Comment by mlosev** [comment]
Да, сейчас можно заходить под всеми активными пользователями в /etc/passwd
Подумаем как лучше добавить ограничение.
Как вариант сейчас можно делать отдельный /etc/passwd только для sprutio - но это не очень удобно

**Comment by AlexeyManikin** [comment]
Самым универсальным было бы решение при котором проверяется, что пользователь состоит в определенной группе и если действительно состоит - разрешать вход. В случае если группа не указана - разрешать вход всем пользователям,

**Comment by f-andrey** [comment]
> Да, сейчас можно заходить под всеми активными пользователями в /etc/passwd

Это кстати ужасное решение, особенно в его текущей реализации, надо либо какую то отдельную базу пользователей для сервисов, коннектор к стандартным LDAP/SQL или ещё как.
Но то что есть, годится только как пруф оф конценпт.

**Comment by shadowlmd** [comment]
Я решил эту проблему следующим образом. В docker-compose.yml для контейнера rpc прописал такие точки монтирования:
```
  volumes:
    - "/sprutio-root:/mnt:rw"
    - "/home:/mnt/home:rw"
    - "./var:/var/spool/fm:rw"
    - "./logs:/var/log/fm:rw"
```
В /sprutio-root/etc копируются файлы passwd, shadow и group только с нужными пользователями и группами. В итоге получаем довольно безопасную конфигурацию: софт не имеет доступа к корневой директории, логиниться могут только те пользователи, которым это разрешено.
